frontend_tests/node_tests/rendered_markdown.js

```diff
@@ -68,7 +68,7 @@ const get_content_element = () => {
     $content.set_find_results('.user-group-mention', $array([]));
     $content.set_find_results('a.stream', $array([]));
     $content.set_find_results('a.stream-topic', $array([]));
-    $content.set_find_results('span.timestamp', $array([]));
+    $content.set_find_results('.timestamp', $array([]));
```

Comment:
There is no `.timestamp` now, right?

Reply:
There is, but it's not rendered. We still need to show the tooltip "couldn't parse the timestamp" or similar, and to add i18n to it, that needs to be done on the frontend.

So, when we add the tooltip, we simultaneously remove the .timestamp class from the element.

Reply:
This was in my implementation, I'm not sure how Tim has rewritten this.

Reply:
Yeah, was just modifying this; the test needs to be rewritten to mainly consider `<time>` tags with invalid content.